tests/Backend/Workspaces/WorkspacesTest.php

```diff
@@ -186,6 +186,7 @@ public function testDropWorkspace($dropOptions)
         $backend->createTable("mytable", ["amount" => ($connection['backend'] === self::BACKEND_SNOWFLAKE) ? "NUMBER" : "VARCHAR"]);
         if ($backend instanceof TeradataWorkspaceBackend) {
             // Teradata: cannot drop workspace if user is logged in
+            $backend->disconnect();
```

Comment:
Hmm to bysme mohli vyřešit v driveru, prostě ukončit všecky sessions když sa volá drop.